plugins/chat/lib/chat_message_updater.rb

```diff
@@ -32,6 +32,7 @@ def update
       @chat_message.save!
       update_uploads(upload_info)
       revision = save_revision!
+      @chat_message.reload
```

Comment:
This is needed because without it `ChatPublisher.publish_edit!` will not have the updated upload AR relations loaded